paper.css

```diff
@@ -9,19 +9,22 @@ body { margin: 0 }
 }
 
 /** Paper sizes **/
-body.A3           .sheet { width: 297mm; height: 419mm }
-body.A3.landscape .sheet { width: 420mm; height: 296mm }
-body.A4           .sheet { width: 210mm; height: 296mm }
-body.A4.landscape .sheet { width: 297mm; height: 209mm }
-body.A5           .sheet { width: 148mm; height: 209mm }
-body.A5.landscape .sheet { width: 210mm; height: 147mm }
+body.A3           .sheet { width: 297mm; height: 419mm;margin-left: auto; margin-right: auto; }
```

Comment:
I'm not sure the benefit of this. Could you pls explain it?
Anyway it seems better to write it in another line:

```css
.sheet { margin-left: auto; margin-right: auto }
```

Reply:
I have open an issue asking for centering the sheets as this PR. https://github.com/cognitom/paper-css/issues/20. I think centering the sheets is great for visual purpose for the clients/users a and I think it should be centered as default. I always for default center all the sheets using this:
```
.sheet {
     margin: 0 auto;
}
```
Think about MS Word, or any other printing tools. Sheets are centered for default. Centering sheets for print is like a standard for the clients/users.